src/_help/hubs/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Hubs
+---
+
+- TOC
+{:toc}
+
+## What is a hub
+
+Hubs are a feature available starting from our paid plans: they allow you to manage collections of API documentation simply and efficiently.
+
+With hubs, your API ecosystem is displayed on a single page, where documentation can be sorted by categories.
+
+Hubs also provide a unified changelog, making it easy to identify all changes across all the documentation it contains at a glance. Search is also global, allowing you to search across all documentation.
+
+## Create a hub
+
+In your dashboard, click on Hubs then on Create a hub.
+
+![](/images/help/create-hub.png)
+
+You will be asked to choose a name for the hub, which will be visible to everyone. The slug is the portion of the URL that you can customize. You can also choose to use a custom domain instead. Finally, you can decide whether this hub is visible to everyone or only to members of your organization.
+
+![](/images/help/hub-creation.png)
+
+## Add documentation to a hub
+
+### Using the webapp
+#### Deploy a new documentation
+From the "Hubs" tab, click on "Create Documentation" to add it to the desired hub. You can then specify the documentation's name, its slug, and its access level (Public or private).
```

Comment:
If I'm correct, there's actually two ways to add a new doc to a hub:
* from `/docs` ("standalone documentation"), which will result in the hub field in the form to be populated with "standalone documentation" and that you can switch to another value
* from `/hubs`. Clicking on the "new doc button" in a hub docs listing will result in having the same form than for the previous case, except that the "hub" field will be populated with the hub name, and cannot be modified"

Reply:
👍 
Modified accordingly, even if my position was to limit amount on information to "what's going on in hub section only".